portfolio/src/main/webapp/index.html

```diff
@@ -8,72 +8,100 @@
     </head>
     <body>
         <div class="menu">
-            <h2>My Portfolio</h2>
             <div class="labels-section">
-                <p>Filter by language:</p>
-                <ul id="labels-list">
+                <p>Filter by language</p>
+                <ul class="labels-list">
                     <li>
-                        <div class="label-shape">
+                        <div class="label">
+                            <div class="label-semi-circle-part"></div>
+                            <div class="label-main-part" style="background-color: green"></div>
                         </div>
                         <p class="label-name">JS</p>
                     </li>
                     <li>
-                        <div class="label-shape">
+                        <div class="label">
+                            <div class="label-semi-circle-part"></div>
+                            <div class="label-main-part" style="background-color: red"></div>
                         </div>
                         <p class="label-name">C++</p>
                     </li>
                     <li>
-                        <div class="label-shape">
+                        <div class="label">
+                            <div class="label-semi-circle-part"></div>
+                            <div class="label-main-part" style="background-color: blue"></div>
                         </div>
                         <p class="label-name">Java</p>
                     </li>
                 </ul>
             </div>
             <div class="sorting-section">
-                <h4>Sorting options</h4>
-                <label for="sort-attribute">Sort-by</label>
-                <select name="sort-attribute">
-                    <option value="empty" disabled selected></option>
-                    <option value="name">Name</option>
-                    <option value="date">Date</option>
-                </select>
-                <input type="radio" name="sort-type" value="asc" />
-                <label for="asc">Ascending</label>
-                <input type="radio" name="sort-type" value="desc" />
-                <label for="desc">Descending</label>
+                <p>Sorting options</p>
```

Comment:
Not a header?

Reply:
Because I think it's better if it's a small font size (design wise).